pkg/controller/migmigration/restore.go

```diff
@@ -280,17 +310,27 @@ func (t *Task) hasRestoreCompleted(restore *velero.Restore) (bool, []string) {
 			restore.Namespace,
 			restore.Name)
 		reasons = append(reasons, message)
+		itemsRestored, totalItems := getRestoreStats(restore)
+		message = fmt.Sprintf(
+			"%s %d out of estimated total of %d objects restored%s",
```

Comment:
If you make the whitespace changes suggested above, the space after "restored" is needed here too.

Reply:
Updated the PR.